FTBFS v1.2.162

When I tried to build tag `v1.2.162` I get an error (gcc, Linux => `-Werror`, `VULKAN_HPP_RUN_GENERATOR=TRUE`):
```
~/Projects/Vulkan-Hpp/VulkanHppGenerator.cpp: In member function ‘std::string VulkanHppGenerator::constructCommandResultGetTwoVectors(const string&, const VulkanHppGenerator::CommandData&, bool, const std::map<long unsigned int, long unsigned int>&) const’:
~/Projects/Vulkan-Hpp/VulkanHppGenerator.cpp:4094:8: error: variable ‘firstVectorParamIt’ set but not used [-Werror=unused-but-set-variable]
 4094 |   auto firstVectorParamIt = vectorParamIndices.begin();
      |        ^~~~~~~~~~~~~~~~~~
cc1plus: all warnings being treated as errors
```
It is better to prepend `auto firstVectorParamIt` by `[[maybe_unused]]` or to move it into the `#if !defined( NDEBUG )` block, where it is solely used. Which approach is preferable? I can prepare a pull request btw.

--- a/VulkanHppGenerator.cpp
+++ b/VulkanHppGenerator.cpp
@@ -4091,11 +4091,10 @@ std::string
 {
   assert( commandData.returnType == "VkResult" );
 
-  auto firstVectorParamIt = vectorParamIndices.begin();
-
   assert( commandData.params[0].type.type == commandData.handle );
 
 #if !defined( NDEBUG )
+  auto firstVectorParamIt = vectorParamIndices.begin();
   auto secondVectorParamIt = std::next( firstVectorParamIt );
   assert( firstVectorParamIt->second == secondVectorParamIt->second );
 #endif

